### DISCUSSION TITLE
The 1.6 PRE-RELEASE 2 of Kade Engine

### DISCUSSION THREAD

**Discussion by Kade-github** [KadeArchive/Kade-Engine#1380] [General]
Fixed:
- Preload Characters not actually preloading anything
- Fixed layering in charting state so you can actually select every character.
- Fixed SM charts to not have random ass holds at the top and actually port over rolls
- You can no longer mash with ghost tapping off and not receive any health decrease.
- Fixed SM replays so they work now
- Pressing 7 no longer lets you die, fixing a bug where if you pressed 7 and died you would be sent to the shadow realm.

<hr /><em>This discussion was created from the release <a href='https://github.com/KadeDev/Kade-Engine/releases/tag/1.6-PRE-RELEASE2'>The 1.6 PRE-RELEASE 2 of Kade Engine</a>.</em>

**Comment by RagdollClash** [comment]
> fixing a bug where if you pressed 7 and died you would be sent to the shadow realm

whomever pressed 7 at the time:
![](https://cdn.discordapp.com/emojis/831359727476146207.gif)

**Reply by BenicioGPXD** [reply]
xd
